src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -1,5 +1,8 @@
 import { h, Component} from 'preact'
 import classNames from 'classnames'
+// import { QRCode }  from 'react-qr-svg'
+import QRCode from 'qrcode.react'
```

Comment:
There is a 6% size increase with this package, which is not huge, but I would still consider splitting this code in a separate bundle to limit size impact.

Reply:
I was thinking this would be part of the lazy loaded cross device bundle. 
Is there anything I will need to set up in the Webpack config to ensure this?

Reply:
Currently we only have the number input validation in the cross device chunk. We can do something similar to what we do [here](https://github.com/onfido/onfido-sdk-ui/blob/fb8cc7cd9b3df2667c17aa15b5e64db02fcf49b5/src/components/PhoneNumberInput/Lazy.js#L36) or refactor that code to include QR

Reply:
OK. Will add a note about this in the implementation ticket. I think making a separate lazy loaded QR code generator like what's being done for phone number input would be the way to go given how we'll be implementing it, i.e. the relevant UI is displayed based on user selection.